src/workerd/io/io-context.h

```diff
@@ -184,6 +186,8 @@ class IoContext_IncomingRequest final {
   kj::Maybe<kj::Own<BaseTracer>> workerTracer;
   kj::Own<IoChannelFactory> ioChannelFactory;
 
+  SpanParent currentUserTraceSpan = nullptr;
```

Comment:
Not sure if the order of workerTracer and currentUserTraceSpan matters here, but I think putting them next to each other and noting that they are supposed to have the same lifetime would be helpful.

Reply:
Well, they don't exactly have the same lifetime, since the span is not created until delivered() is called. I think the ordering is not that important TBH.